site/components/Button/Button.tsx

```diff
@@ -32,7 +32,7 @@ type BaseProps = {
   >;
 
 type WithAnchor = {
-  as?: 'a';
+  as?: 'a' | 'span';
```

Comment:
Should this be on the `WithoutAnchor` type? Otherwise the types say it's valid to create a span with an `href` attribute.

Reply:
makes sense!